src/chainer.js

```diff
@@ -8,11 +10,12 @@ function chainer(functions) {
     let result = x;
 
     for (const f of functions) {
-      result = f(result);
+      if (typeof f === 'function') {
+        result = f(result);
+      }
```

Comment:
Although it's good to check if each item in the functions array is indeed a function, there should be some error handling mechanism in place if it's not. For example, you can throw an error or ignore the non-function item.